src/Server/Grabber/Grabber.ts

```diff
@@ -29,6 +29,26 @@ export class Grabber {
             });
         });
     }
+ public static getCourseInSession(course: CourseInfo): Promise<string[]> {
```

Comment:
Add some doc string for this function, as the name does not clearly explain what it is doing